cmake/libROMConfig.cmake.in

```diff
@@ -0,0 +1,19 @@
+## Some copyright problably goes here
```

Comment:
I suppose you can just copy the header from the main libROM/CMakeLists.txt file. @chldkdtn can you please confirm?

Reply:
Correct. @JacobLotz, you can copy it from the header of the main libROM/CMakeLists.txt, i.e., 

###############################################################################
#
#  Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
#  and other libROM project developers. See the top-level COPYRIGHT
#  file for details.
#
#  SPDX-License-Identifier: (Apache-2.0 OR MIT)
#
###############################################################################

Reply:
Check!